lib/active_merchant/billing/gateways/adyen_checkout.rb

```diff
@@ -348,21 +369,24 @@ def cvv_result_from(response)
         CVC_MAPPING[response['additionalData']['cvcResult'][0]] if response.dig('additionalData', 'cvcResult')
       end
 
-      def refund?(action)
-        action == "refund"
+      def should_use_pal_endpoint?(action)
```

Comment:
[non-blocking] `use_pal_endpoint?` :) because less chars is better

Reply:
done 👍